src/components/weather-selector/weather-selector.html

```diff
@@ -0,0 +1,35 @@
+<!-- Generated template for the WeatherSelectorComponent component -->
+<ion-content padding>
+  <h1>Select weather conditions</h1>
+  <div class="mes-divider"></div>
+  <ion-grid>
+    <ion-row>
+      <ion-col>
+        <ion-row *ngFor="let weather of weatherConditions; let i = index" class="conditionsList">
+          <ion-col *ngIf="!isEven(i)">
```

Comment:
instead of using isEvent you can use **odd**, but it's up to you. I'm fine with isEven method either
You can read more about it here:
https://angular.io/guide/structural-directives#inside-ngfor

Reply:
Nice! I didn't know that, I'm going to change it over